src/main/java/de/themorpheus/edu/taskservice/controller/solution/SolutionController.java

```diff
@@ -6,12 +6,20 @@
 import de.themorpheus.edu.taskservice.database.repository.solution.SolutionRepository;
 import de.themorpheus.edu.taskservice.util.ControllerResult;
 import de.themorpheus.edu.taskservice.util.Error;
+import java.util.ArrayList;
+import java.util.List;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ClassPathScanningCandidateComponentProvider;
+import org.springframework.core.type.filter.TypeFilter;
 import org.springframework.stereotype.Controller;
 
 @Controller
 public class SolutionController {
 
+	private static List<SolutionInterface> solutionInterfaces = new ArrayList<>();
```

Comment:
This list does contain instances of these interfaces - not the interfaces themselfes